Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -231,14 +231,15 @@ def return_entry(message, existing_incident=None):
 
 
 def close_new_incident_and_link_to_existing(new_incident, existing_incident, similarity):
-    res = demisto.executeCommand("CloseInvestigationAsDuplicate", {
-        'duplicateId': existing_incident['id']})
-
-    if is_error(res):
-        return_error(res)
-    message = 'Duplicate incidents found: #{}, #{} with similarity of {:.1f}%. '.format(new_incident['id'],
-                                                                                        existing_incident['id'],
-                                                                                        similarity * 100)
+    if demisto.args().get('closeAsDuplicate', 'true') == 'true':
+        res = demisto.executeCommand("CloseInvestigationAsDuplicate", {
+            'duplicateId': existing_incident['id']})
+
+        if is_error(res):
+            return_error(res)
+    formatted_incident = format_similar_incident(existing_incident, similarity)
+    message = tableToMarkdown("Duplicate incident found with similarity {:.1f}%".format(similarity * 100),
+                              formatted_incident)
     message += 'This incident will be closed and linked to #{}.'.format(existing_incident['id'])
```

Comment:
change the message based on `demisto.args().get('closeAsDuplicate', 'true') == 'true'`